dev-cpp/cpptoml/cpptoml-0.1.1.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Header-only library for parsing TOML"
+HOMEPAGE="https://github.com/skystrife/cpptoml"
+SRC_URI="https://github.com/skystrife/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="libcxx examples"
+
+RDEPEND="libcxx? ( sys-libs/libcxx )"
```

Comment:
This looks a bit suspicious. What is this for and why is it here? It should be done based on `tc-get-cxx-stdlib` if it's appropriate to have at all...

Reply:
Not sure, the original package has a flag for it: 

https://github.com/skystrife/cpptoml/blob/fededad7169e538ca47e11a9ee9251bc361a9a65/CMakeLists.txt#L29

So presumably it's important to let the package know. I'll try to do it the way you mentioned though.